etl/steps/data/garden/tourism/2023-05-05/unwto.meta.yml

```diff
@@ -14,177 +11,313 @@ dataset:
         display:
+          name: International arrivals for personal reasons
           numDecimalPlaces: 0
+
       ou_to_de_ov_vi_to:
         title: Outbound departures (tourists)
-        unit: departures
+        unit: ''
+        description: >
+          Tourist departures refer to the number of trips by people who travel abroad and stay overnight. Departures data measure the flows of resident visitors leaving the country of reference.
         display:
+          name: International tourist departures
           numDecimalPlaces: 0
+
       ou_to_de_sa_da_vi_ex:
         title: Outbound departures (same day)
-        unit: departures
+        unit: ''
+        description: >
+          Departures by same-day visitors refer to the number of trips by people who travel abroad and come back on the same day. Departures data measure the flows of resident visitors leaving the country of reference.
         display:
+          name: International departures by same-day visitors
           numDecimalPlaces: 0
+
       ou_to_ex_tr:
-        title: Outbound tourism-expenditure
+        title: Outbound tourism expenditure
         unit: US dollars
         short_unit: $
+        description: >
+          Expenditures associated with international visitors are traditionally categorized under the travel item of the Balance of Payments (BOP). In the case of outbound tourism, those expenditures associated with resident visitors are registered as “debits” in the BOP and refers to “travel expenditure”.
+
+
+          It's important to note that BOP estimates also include expenditures related to other types of travelers, such as long-term students, patients, border workers, and seasonal workers, which can be substantial in some countries.
         display:
+          name: International tourist expenditure abroad
           numDecimalPlaces: 0
+
       to_in_av_le_of_st:
         title: Average length of stay
         unit: nights
+        description: >
+          This indicator refers to the duration of trips abroad by outbound visitors (expressed as the number of days) and reflects the total average using border surveys and/or household information.
         display:
           numDecimalPlaces: 1
+
       inb_tour_region:
         title: International tourist arrivals by region
         unit: ''
+        description: >
+          The underlying premise behind this indicator is that the country linked to the arrival should correspond to the individual's country of residence. However, it is worth noting that certain countries deviate from UNWTO recommendations and classify their nationals residing abroad as a distinct category, rather than considering them as residents of those countries.
         display:
           numDecimalPlaces: 0
+
       ou_to_de_to_de_per_1000:
         title: Outbound departures (total) per 1000
```

Comment:
These variables are sometimes titled "per 1000" and sometimes "per 1000 people." Perhaps we should standardize that.

It may also be worth mentioning in the descriptions what "per 1000 people" means, i.e. 1000 people in the population.